.github/workflows/publish.yml

```diff
@@ -56,7 +56,7 @@ jobs:
 
       # See https://github.com/actions/runner/issues/2033
       - name: Fix git safe.directory in container
-        run: mkdir -p /home/runner/work/_temp/_github_home && printf "[safe]\n\tdirectory = /github/workspace" > /home/runner/work/_temp/_github_home/.gitconfig
+        run: mkdir -p /home/runner/work/_temp/_github_home && printf "[safe]\n\tdirectory = /github/workspace/__repo__" > /home/runner/work/_temp/_github_home/.gitconfig
```

Comment:
2 things:

1. Why not place this right after `checkout`?
2. How about making that `__repo__` path a global env value or something so you don't need to repeat it and avoid this mistake in the future?

Reply:
(1) ... makes sense. Done in 6598983762e4a24637395dc531ae3486d995ac4d. I also removed the mkdir since if we have a checkout in `/home/runner/work/_temp/_github_home/__repo__` then we are guaranteed for `/home/runner/work/_temp/_github_home` to exist (right?).

(2) I agree with your intuition, but I think it's more trouble than it's worth for this PR.

Reply:
> (right?)

Wrong. #1832.